ci/teams/rhtap-servicerelease/external-secrets/pyxis-secret.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: pyxis-staging-secret
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  dataFrom:
+    - extract:
+        key: "staging/release/pyxis/certificate"
```

Comment:
This is a directory. This works? So it injects both the key and certificate and knows to decode them?

Reply:
okay cool